### ISSUE TITLE
`xc2c256.map` for flashing urukul rev 1.4

### ISSUE THREAD

**Issue by lriesebos** [sinara-hw/Urukul#62]
When flashing an urukul rev 1.4 (without xilinx installed) I get an error message that I need the file `xc2c256.map` instead of the provided `xc2c128.map` available on the wiki. see the message below. Where can I find this file, or am I doing something else wrong? @gkasprow @jbqubit 

```
leon@tokachi:~/urukul$ xc3sprog -c jtaghs2 -m map -v urukul.jed:w
XC3SPROG (c) 2004-2011 xc3sprog project $Rev$ OS: Linux
Free software: If you contribute nothing, expect nothing!
Feedback on success/failure/enhancement requests:
	http://sourceforge.net/mail/?group_id=170565 
Check Sourceforge for updates:
	http://sourceforge.net/projects/xc3sprog/develop

Using built-in device list
Using built-in cable list
Cable jtaghs2 type ftdi VID 0x0403 PID 0x6014 Desc "Digilent USB Device" dbus data e8 enable eb cbus data 00 data 60
Using Libftdi, Using JTAG frequency   6.000 MHz from undivided clock
JTAG chainpos: 0 Device IDCODE = 0x16d4e093	Desc: XC2C256_FT256
Mapfile map/xc2c256.map not found: No such file or directory
No mapfile available -- only bitfile (not JEDEC) operations will be supported.
Reading style BIT
Ignoring unknown field '1'
Ignoring unknown field '1'
Ignoring unknown field '1'
Ignoring unknown field '1'
Ignoring unknown field '7'
Ignoring unknown field '1'
Ignoring unknown field '1'
Ignoring unknown field '9'
Ignoring unknown field '1'
Ignoring unknown field '1'
Ignoring unknown field '
'
Unknown error
Probably no Bitfile, aborting
USB transactions: Write 6 read 3 retries 3
```

OS: Ubuntu 20.04 (xc3sprog installed from Ubuntu repositories)

**Comment by jordens** [comment]
Are you allowed to redistribute that file?

**Comment by lriesebos** [comment]
closed by 945f099
